packages/nimble-components/karma.conf.js

```diff
@@ -49,7 +49,8 @@ module.exports = config => {
             'karma-source-map-support',
             'karma-sourcemap-loader',
             'karma-chrome-launcher',
-            'karma-firefox-launcher'
+            'karma-firefox-launcher',
+            'karma-webkit-launcher'
```

Comment:
Similarly, can we make sure to use playwright's webkit copy: https://github.com/google/karma-webkit-launcher#manually-define-playwright-executable

Reply:
Wonder if that was part of the CI issues? Maybe they have their own webkit version configured and we aren't picking up playwrights?

Reply:
Added the environment variables to force Playwright for WebKit headless and headed.

As discussed the CI issues (hang) when running the WebKit tests still occurs.  
[Here](https://github.com/ni/nimble/actions/runs/4236125814/jobs/7360568605) is an example build that ran the WebKit tests (the test run finished, with the 6 pre-existing test failures), and then hung later on after Angular tests and before the Blazor tests, so I cancelled it after a few minutes.